Documentation/tutorial/docfx.exe_user_manual.md

```diff
@@ -184,8 +184,8 @@ content                  | Contains all the files to generate documentation, inc
 resource                 | Contains all the resource files that conceptual and metadata files dependent on, e.g. image files. `name-files` file mapping with several ways to define it, as to be described in **Section4**.
 overwrite                | Contains all the conceputal files which contains yaml header with `uid` and is intended to override the existing metadata `yml` files. `name-files` file mapping with several ways to define it, as to be described in **Section4**.
 globalMetadata           | Contains metadata that will be applied to every file, in key-value pair format. For example, you can define `"_appTitle": "This is the title"` in this section, and when applying template `default`, it will be part of the page title as defined in the template.
-globalMetadataFile       | Specify a JSON file path containing globalMetadata settings, as similar to `{"globalMetadata":{"key":"value"}}`. It overrides the globalMetadata settings from the config file.
-fileMetadataFile         | Specify a JSON file path containing fileMetadata settings, as similar to `{"fileMetadata":{"key":"value"}}`. It overrides the fileMetadata settings from the config file.
+globalMetadataFiles      | Specify a list of JSON file path containing globalMetadata settings, as similar to `{"globalMetadata":{"key":"value"}}`. It overrides the globalMetadata settings from the config file, and `globalMetadataFiles` from command line will override it from the config file.
+fileMetadataFiles        | Specify a list of JSON file path containing fileMetadata settings, as similar to `{"fileMetadata":{"key":"value"}}`. It overrides the fileMetadata settings from the config file, and `fileMetadataFiles` from command line will override it from the config file.
```

Comment:
yes, this table is about build command option, and it can not set `fileMetadata` currently.